plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -58,6 +64,9 @@ public TestingPostgreSqlServer()
                 .withUsername(USER)
                 .withPassword(PASSWORD)
                 .withCommand("postgres", "-c", "log_destination=stderr", "-c", "log_statement=all");
+        if (shouldExposeFixedPort) {
+            configureFixedExposedPortForLocalEnv(dockerContainer);
```

Comment:
The logic inside that method no longer needs to be dependent on `System.getenv("CONTINUOUS_INTEGRATION")`

It can, however, `checkState( System.getenv("CONTINUOUS_INTEGRATION") == null, "Exposing fixed ports should not be used in regular test code. This could break parallel test execution locally. This method is supposed to be invoked from local development helpers only, hence it should never be run on CI")`